mesoshttp/client.py

```diff
@@ -340,6 +340,8 @@ def __init__(
         :type frameworkWebUI: str
         :param max_reconnect: number of reconnection retries when connection fails
         :type max_reconnect: int  defaults to 3
+        :param connection_timeout: sets a timeout for scheduler connection loop
```

Comment:
You define param connection_timeout here, but it is not in def params if init.
You could add param with a default value to None.
Type/param are here to define input params.

If you only want to declare instance var, to be set by app, then just remove type/param def here.

Reply:
Nice catch! thanks for pointing that out, i have added connection_timeout as an input parameter for init.